android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java

```diff
@@ -266,10 +267,14 @@ private static void transformJson(
      * MOAR COLORZ PLS!
      */
     private static String getColorFrom(String roomName) {
-        String roomColor = DEFAULT_ROOM_COLOR;
+        if (roomName == null) {
+            return DEFAULT_ROOM_COLOR;
+        }
+
+        String roomColor = ROOMS_TO_COLORS.get(roomName.toLowerCase());
```

Comment:
While we're here you might also want to do `roomName.trim().toLowerCase()`, but again, nitpicking...